lib/wasm-async/AsyncWasmLoadingRuntimeModule.js

```diff
@@ -52,38 +52,59 @@ class AsyncWasmLoadingRuntimeModule extends RuntimeModule {
 				runtime: chunk.runtime
 			}
 		);
+
+		const loader = this.generateLoadBinaryCode(wasmModuleSrcPath);
+		const fallback = [
+			`.then(${runtimeTemplate.returningFunction("x.arrayBuffer()", "x")})`,
+			`.then(${runtimeTemplate.returningFunction(
+				"WebAssembly.instantiate(bytes, importsObj)",
+				"bytes"
+			)})`,
+			`.then(${runtimeTemplate.returningFunction(
+				"Object.assign(exports, res.instance.exports)",
+				"res"
+			)})`
+		];
+		const getStreaming = () => {
+			const concat = (/** @type {string[]} */ ...text) => text.join("");
+			return [
+				`var req = ${loader};`,
+				`var fallback = ${runtimeTemplate.returningFunction(Template.asString(["req", Template.indent(fallback)]))};`,
+				concat(
+					"return req.then(",
+					runtimeTemplate.basicFunction("res", [
+						`if (typeof WebAssembly.instantiateStreaming === "function") {`,
+						Template.indent([
+							"return WebAssembly.instantiateStreaming(res, importsObj)",
```

Comment:
Looks like we miss `.then((res) => (Object.assign(exports, res.instance.exports)));`

Reply:
Before:
```js
if (typeof WebAssembly.instantiateStreaming === 'function') {
/******/ 				return WebAssembly.instantiateStreaming(req, importsObj)
/******/ 					.then((res) => (Object.assign(exports, res.instance.exports)));
/******/ 			}
```

Now:
```js
if (typeof WebAssembly.instantiateStreaming === "function") {
/******/ 					return WebAssembly.instantiateStreaming(res, importsObj)
/******/ 						.catch((e) => {
/******/ 							if(res.headers.get("Content-Type") !== "application/wasm") {
/******/ 								console.warn("`WebAssembly.instantiateStreaming` failed because your server does not serve wasm with `application/wasm` MIME type. Falling back to `WebAssembly.instantiate` which is slower. Original error:\n", e);
/******/ 								return fallback();
/******/ 							}
/******/ 							throw e;
/******/ 						});
/******/ 				}
```